README.md

```diff
@@ -1,64 +1,59 @@
-
-## Instructions
-
-Open the network inspector in Chrome or Firefox and ensure 'preserve log' is ticked. Run your manual or functional test.
-
-After the test is complete, right-click on one of the requests in the network inspector and save the log as a HAR file. Save this file in the root directory of this project as `requests.har` or alternatively use the path to this HAR file in the `HAR_FILEPATH` environment variable when running locust.
+The benchmark consumes a requests JSON file that contains a list of HTTP requests. This can either be created from scratch or generated from a HAR file. Example requests files can be found in the `requests` folder.
 
 To run a benchmark, use:
 
 ```bash
-pipenv install
-pipenv run ./run.sh <SCHEMA_NAME> <HOST: Optional>
+pipenv run ./run.sh <REQUESTS_JSON> <HOST: Optional>
 ```
 e.g.
-```
-pipenv run ./run.sh test_mutually_exclusive
+```bash
+pipenv run ./run.sh requests/test_checkbox.json
 ```
 
-This will run 1 minute of locust requests with 1 user and no wait time between requests.
-
-The output file can be found in `output_requests_processed.csv`
-
-If you'd like to run locust on its own, you can use the following commands
+This will run 1 minute of locust requests with 1 user and no wait time between requests. The output file can be found in `output_requests_processed.csv`.
 
 For the web interface:
 
 ```bash
-SCHEMA_NAME=test_hub_and_spoke HOST=http://localhost:5000 pipenv run locust -f runner_benchmark/har_replay_test.py
-```
-
-For a command line interface ( with output CSV file ):
-
-```bash
-SCHEMA_NAME=test_hub_and_spoke HOST=http://localhost:5000 pipenv run locust -f runner_benchmark/har_replay_test.py --no-web -t 1m -c 100 -r 10 --csv=output
+REQUESTS_JSON=requests/test_checkbox.json HOST=http://localhost:5000 pipenv run locust
 ```
 
 ## Configuration
 
 The following environment variables can be used to configure the locust test:
 
-- `HAR_FILEPATH` - The filepath of the HAR file relative to the base project directory
-  - Defaults to `requests.har`
-- `SCHEMA_NAME` - The name of the schema being tested.
+- `REQUESTS_JSON` - The filepath of the requests file relative to the base project directory
+  - Defaults to `requests.json`
 - `HOST` - The host name the benchmark is run against.
 - `USER_WAIT_TIME_MIN_SECONDS` - The minimum delay between each user's GET requests
   - defaults to zero
 - `USER_WAIT_TIME_MAX_SECONDS` - The maximum delay between each user's GET requests
   - defaults to zero
 
+## Generating a requests file
+
+Open the network inspector in Chrome or Firefox and ensure 'preserve log' is ticked. Run your manual or functional test.
```

Comment:
Would be good to include the fact that the session should not be included here. That's caused some confusion in the past.